app/controllers/account/my_tenants_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Account::MyTenantsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_if_admin, only: %i[edit update]
+  before_action :current_tenant
```

Comment:
remove this before_action 

just call current_tenant (I think we already have such method in application_controller)